secure/storage/vault/src/lib.rs

```diff
@@ -372,9 +365,9 @@ impl Client {
     /// Returns whether or not the vault is unsealed (can be read from / written to). This can be
     /// queried without authentication.
     pub fn unsealed(&self) -> Result<bool, Error> {
-        let resp = ureq::get(&format!("{}/v1/sys/seal-status", self.host))
-            .timeout_connect(TIMEOUT)
-            .call();
+        let request = ureq::get(&format!("{}/v1/sys/seal-status", self.host));
+        let resp = self.upgrade_request_without_token(request).call();
+        println!("{:?}", resp.synthetic_error());
```

Comment:
Same here, also worth considering using these errors if they exist... but that's another PR